src/Compatibility/Core/src/Android/AppCompat/Platform.cs

```diff
@@ -698,7 +698,7 @@ void UpdateBackgroundColor()
 		internal static int GenerateViewId()
 		{
 			// getting unique Id's is an art, and I consider myself the Jackson Pollock of the field
-			if ((int)Forms.SdkInt >= 17)
+			if (OperatingSystem.IsAndroidVersionAtLeast(17))
```

Comment:
@Redth the minimum API level for dotnet/runtime and xamarin/xamarin-android is 21.

Any of these checks for old API levels, should I remove them completely? Just assume they are 21 or higher?

Reply:
Discussed on a call, I'll remove anything checking older than 21.